internal/operators/manager.go

```diff
@@ -463,13 +448,25 @@ func EnsureLVMAndCNVDoNotClash(cluster *common.Cluster, openshiftVersion string,
 		}
 	}
 
-	if lvmEnabled && cnvEnabled {
+	if !lvmEnabled || !cnvEnabled {
+		return nil
+	}
+
+	// Openshift version greater or Equal to 4.12.0 support cnv and lvms
+	if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmsMinOpenshiftVersion4_12, openshiftVersion); !isGreaterOrEqual {
```

Comment:
there is a mix(ODF + LVM) in the error message - not your code but please fix it
should be "return errors.Errorf("Currently, you can not install Logical Volume Manager operator at the same time as Virtualization operator.")"

Reply:
changed